src/app/api/api.py

```diff
@@ -245,17 +244,32 @@ async def get_timestamps(
     status_code=HTTP_200_OK,
     responses={HTTP_404_NOT_FOUND: {"description": "Collection not found"}},
     tags=[_TAG_COLLECTIONS],
-    description=f"""Returns the average ratio of non-empty properties for the chosen collection.
-    For certain properties, e.g. `properties.cclom:title`, the ratio of
-    elements which miss this entry compared to the total number of entries is calculated.
-    A missing entry may be `properties.cclom:title = null`. Not all properties are considered here.
-    The overall score is the average of all these ratios.
-    The queried properties are:
-    `{field_names_used_for_score_calculation(aggs_collection_validation)
-      + field_names_used_for_score_calculation(aggs_material_validation)}`.
-    """,
+    summary="The average ratio of non-empty properties for the chosen collection",
 )
 async def score(*, node_id: uuid.UUID = Depends(node_ids_for_major_collections)):
+    """
+    Returns the average ratio of non-empty properties for the chosen collection.
+
+    For certain properties, e.g. `properties.cclom:title`, the ratio of elements which miss this entry compared to the
+    total number of entries is calculated. A missing entry may be `properties.cclom:title = null`. Not all properties
+    are considered here. The overall score is the average of all these ratios.
+
+    The queried fields (all as keywords) are:
+     - properties.cm:title
+     - properties.cclom:general_keyword
+     - properties.cm:description
+     - properties.ccm:educationalcontext
+     - properties.cclom:title
+     - properties.ccm:oeh_lrt
+     - properties.ccm:taxonid
+     - properties.ccm:wwwurl
+     - properties.ccm:oeh_publisher_combined
+     - properties.cclom:general_description
+     - i18n.de_DE.ccm:educationalintendedenduserrole
+     - properties.ccm:educationalcontext
+
+    Fixme: investigate why we list cm:title and cclom:title here.
```

Comment:
We need both a title of the collection, e.g., chemistry, as well as a title for the material, e.g. "CO2 Zyklus" to have a quality material. Therefore both collection and material title are queried.

Reply:
rewrote the whole docstring with more details and remaining open questions.